static/js/components/CouponNotificationDialog.js

```diff
@@ -40,43 +37,25 @@ const CouponNotificationDialog = (
 
   let title, message;
   if ( contentType === COUPON_CONTENT_TYPE_PROGRAM ) {
-    if ( amountType === COUPON_AMOUNT_TYPE_PERCENT_DISCOUNT ) {
-      title = `Coupon applied: ${amount.times(100).toDecimalPlaces(0).toString()}% off each course!`;
-      message = <p>
-        This coupon gives <strong>a discount
-        of { amount.times(100).toDecimalPlaces(0).toString() }% off</strong> the price
-        of <strong>each</strong> course in { programName }.
-      </p>;
-    } else if ( amountType === COUPON_AMOUNT_TYPE_FIXED_DISCOUNT ) {
-      title = `Coupon applied: $${amount} off each course!`;
-      message = <p>
-        This coupon gives <strong>a discount
-        of ${ amount.toString() } off</strong> the price
-        of <strong>each</strong> course in { programName }.
-      </p>;
-    }
+    title = `Coupon applied: ${makeAmountMessage(coupon)} each course!`;
+    message = <p>
+      This coupon gives <strong>a discount
+      of {makeAmountMessage(coupon)}</strong> the price
+      of <strong>each</strong> course in { programName }.
+    </p>;
```

Comment:
This is also going to change in the fixed price PR. In that PR I'm not sure there's enough that will get shared to make it worth sharing (though we can decide that in that PR)